numpy/core/code_generators/generate_numpy_api.py

```diff
@@ -201,9 +204,10 @@ def do_generate_api(targets, sources):
 
     for name, val in types_api.items():
         index = val[0]
-        internal_type =  None if len(val) == 1 else val[1]
+        internal_type = None if len(val) <= 1 else val[1]
```

Comment:
not your fault of course, but this is a very ugly way of specifying arguments :(

Reply:
Yes, that annoys me too, but I didn't see a way to change it without refactoring the whole thing.